blog/blog_server/server.go

```diff
@@ -121,7 +121,7 @@ func (*server) UpdateBlog(ctx context.Context, req *blogpb.UpdateBlogRequest) (*
 
 	// create an empty struct
 	data := &blogItem{}
-	filter := bson.NewDocument(bson.EC.ObjectID("_id", oid))
+	filter := bson.M{"_id": oid}
```

Comment:
why not filter := primitive.M{"_id": oid} instead? bson.M is from another package